usaspending_api/settings.py

```diff
@@ -43,6 +43,16 @@
 # Define local flag to affect location of downloads
 IS_LOCAL = True
 
+# How to handle downloads locally
+# True: process it right away by the API;
+# False: leave the message in the local file-backed queue to be picked up and processed by the bulk-download container
+RUN_LOCAL_DOWNLOAD_IN_PROCESS = (
```

Comment:
Simplify the logic, and make it a one-liner.
Also the conditional was switched to make false the default.

```suggestion
RUN_LOCAL_DOWNLOAD_IN_PROCESS = os.environ.get("RUN_LOCAL_DOWNLOAD_IN_PROCESS", "").lower() in ["true", "1", "yes"]
```

Reply:
@tony-sappe it looks cleaner but I don't think `False` as the default will go over well with the team, and will break unit tests. 

Unit Tests: (I had it this way originally, and found this out) are performing checks on the response to the download endpoint and therefore require that the download get processed right-away, in-process of the test session. Otherwise the download request gets enqueued and not processed at all, and the test fails (e.g. "total_rows" = None).

Devs: Most people probably are not currently running docker-based local dev envs, and those that are may not always run _both_ the api and bulk downloader containers (maybe just the API and/or DB). So where they are expecting a download to fire off the download code immediately, they'll sit there and wait (since it will just put it in the queue and wait for _something_ (that they're not running) to pick it up.

Reply:
good points

```python
RUN_LOCAL_DOWNLOAD_IN_PROCESS = os.environ.get("RUN_LOCAL_DOWNLOAD_IN_PROCESS", "").lower() not in ["false", "0", "no"]
```

Reply:
Done.